plaso/lib/binary.py

```diff
@@ -28,44 +29,44 @@ def ByteStreamCopyToString(byte_stream, codepage='utf-8'):
   """Copies a UTF-8 encoded byte stream into a Unicode string.
 
   Args:
-    byte_stream: A byte stream containing an UTF-8 encoded string.
-    codepage: The codepage of the byte stream.
+    byte_stream (bytes): byte stream containing an UTF-8 encoded string.
+    codepage (Optional[str]): codepage of the byte stream.
 
   Returns:
-    A Unicode string.
+    str: Unicode string.
   """
   try:
-    string = byte_stream.decode(codepage)
+    string = codecs.decode(byte_stream, codepage)
   except UnicodeDecodeError:
     logging.warning(
         'Unable to decode {0:s} formatted byte stream.'.format(codepage))
-    string = byte_stream.decode(codepage, errors='ignore')
+    string = codecs.decode(byte_stream, codepage, errors='ignore')
 
   string, _, _ = string.partition('\x00')
```

Comment:
Wasn't split faster? Maybe change to `string = string.split('\x00')[0]`

Reply:
Split() is only faster for large strings, I think this is better as partition()